internal/cmd/egctl/version.go

```diff
@@ -48,6 +49,8 @@ func NewVersionCommand() *cobra.Command {
 
 	versionCommand.PersistentFlags().StringVarP(&output, "output", "o", yamlOutput, "One of 'yaml' or 'json'")
 
+	versionCommand.PersistentFlags().BoolVarP(&client, "client", "c", false, "If only output client version")
```

Comment:
`client` is not a good name, specifically should not use `c` for short.

what about add `--remote` with default value `true`?

Reply:
Good idea. Thanks!